lib/src/base-styles/_nx-read-only.scss

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+ @import '../scss-shared/nx-variables';
+ @import '../scss-shared/nx-text-helpers';
+ @import '../scss-shared/nx-container-helpers';
+
+.nx-read-only {
+  @include container-vertical;
+  margin: 0 0 $nx-spacing-xl 0;
```

Comment:
No top margin?

Reply:
> No top margin?

I used the exact same settings as we use for `.nx-form-group` which does not specify top margin.